### PR TITLE
feat: Idp 1069 adeguare apim per portale merchant

### PR DESCRIPTION
**Pull request by frtrinca**
### Type of changes

- [x] Add new resources
- [ ] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [x] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [x] No

#### Has This Been Tested?

- [x] Yes
- [ ] No

### Other information
Target: 
```

```

#### How Has This Been Tested?

- [ ] Unit Test Suite
- [ ] Integration Test Suite
- [x] Api Tests
- [ ] Load Tests

[Link to test results](https://pagopa.atlassian.net/browse/RTD-XXXX)

---

### CODE REVIEW

File: src/domains/idpay-common/03_database.tf [hunk 1]
```diff
@@ -487,6 +487,9 @@ locals {
         }, {
         keys   = ["trxChargeDate"]
         unique = false
+        }, {
```
Comment (GiovanniMancini) [inline src/domains/idpay-common/03_database.tf:490]: @frtrinca Could you move index update in another PR? It's a different terraform domain and cross-domain apply could rapidly get into a mess

Comment (dariopelliccioli) [reply src/domains/idpay-common/03_database.tf]: index moved in another PR
